packages/Sortable/src/Sortable.js

```diff
@@ -63,7 +63,12 @@ const Sortable = ({ children, onChange, hasNumbers, hasZebraStripes, onRemove, .
   };
 
   return (
-    <DragDropContext onDragEnd={handleDragEnd} onDragStart={handleDragStart} onDragUpdate={handleDragUpdate}>
+    <DragDropContext
+      onDragEnd={handleDragEnd}
+      onDragStart={handleDragStart}
+      onDragUpdate={handleDragUpdate}
+      liftInstruction={I18n.t("sortable.aria_description")}
```

Comment:
This is the new prop for `react-beautiful-dnd`?

Reply:
Yes, without this prop it does not announce `Press space to lift` when you tab over the items.